packages/swapper/src/swappers/zrx/ZrxApprovalNeeded/ZrxApprovalNeeded.ts

```diff
@@ -28,7 +28,7 @@ export async function ZrxApprovalNeeded(
       return { approvalNeeded: false }
     }
 
-    const accountNumber = bnOrZero(quote.sellAssetAccountId).toNumber()
+    const accountNumber = bnOrZero(quote.sellAssetAccountNumber).toNumber()
```

Comment:
Do we still need to `bnOrZero().toNumber()` this since it should already be a number according to `quote` typings?